modules/pivot-table/main/index.coffee

```diff
@@ -43,42 +71,44 @@ class PivotTable extends hx.EventEmitter
     @tableHead = @table.append('thead')
     @tableBody = @table.append('tbody')
 
+    # Create the re-usable views
+    [@tableHeadView, @tableBodyView] =
+      createTableView(this, @tableHead, @tableBody)
+
+    if @options.data?
+      @data(@options.data)
```

Comment:
Allow setting of data in the options, can be done in the standard and fluid versions